internal/pkg/prometheus/prometheus.go

```diff
@@ -36,15 +36,23 @@ func ResetTargetSize() {
 	targetSize.Reset()
 }
 
+const (
+	// XPrometheusHeader included in all requests. It informs exporters about its timeout.
+	XPrometheusHeader = "X-Prometheus-Scrape-Timeout-Seconds"
```

Comment:
Nit: should we use `XPrometheusScrapeTimeoutHeader` as constant name? It is quite a bit long but it is also more descriptive.